carma-messenger-core/cpp_message/src/cpp_message.cpp

```diff
@@ -200,6 +200,8 @@ namespace cpp_message
             }
         }
         else if(msg->message_type=="PSM"){
+            auto current_time = this->now();
+            RCLCPP_DEBUG_STREAM(get_logger(), "Incoming PSM at timestamp sec:"<<current_time.seconds()<<" nanosec: "<<current_time.nanoseconds());
```

Comment:
seconds() and nanoseconds() functions are both absolute UTC measures. Probably best to pick one to avoid confusion. Nanoseconds will technically give the most accurate value by avoiding floating point rounding, but if you would do that anyway before analysis it defeats the benefit and seconds could be used instead.